OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs

```diff
@@ -67,5 +67,15 @@ public partial class ClassroomService : IClassroomService
 
 			return storageClassrooms;
 		});
+
+		public ValueTask<Classroom> RetrieveClassroomById(Guid classroomId) =>
+		TryCatch(async () =>
+		{
+			Classroom storageClassroom =
+				await this.storageBroker.SelectClassroomByIdAsync(classroomId); await this.storageBroker.SelectClassroomByIdAsync(classroomId);
```

Comment:
await this.storageBroker.SelectClassroomByIdAsync(classroomId);  is called twice here

Reply:
I think its was because of conflict, its removed now.